### PR TITLE
MERC-3746 cornerstone schema change

### PR DESCRIPTION
**Pull request by kchu93**
#### What?

Cornerstone - Update Schema labels and organization;
Updated the labels and organization of cornerstone of schema.json entries. 

#### Why?
In preparation for launching Store Design, the merchandising team performed an audit on the cornerstone settings that are available finding a large number of usability gaps. As a short-term solution, the team made the following changes focusing on providing a better legibility of the setting.

- Update labels on all entries to use updated, consistent language
- Provide better organization by creating new groupings of settings
- User clarity and consistency

#### Tickets / Documentation

Add links to any relevant tickets and documentation.

- [MERC-3746](https://jira.bigcommerce.com/browse/MERC-3746)
- [Store Design Spread Sheet](https://docs.google.com/spreadsheets/d/1cgpYVzrd2X1MAOvPKJjkdAX5iqn7bTmeGKTHhklu8UA/edit?ts=5b648b18#gid=524958021)

### CODE REVIEW

File: schema.json [hunk 1]
```diff
@@ -1108,70 +1148,113 @@
...
+            "label": "Burst"
+          }
+        ]
       },
       {
         "type": "text",
-        "label": "Product sale price label",
-        "id": "pdp-sale-price-label"
+        "label": "Product price label (sale)",
+        "force_reload": true,
```
Comment (junedkazi) [inline schema.json]: @pedelman @kchu93 thank you for doing this as part of this PR.

File: schema.json [hunk 2]
```diff
@@ -1108,70 +1148,113 @@
...
-        "id": "pdp-price-label"
+        "force_reload": true,
+        "id": "pdp-price-label",
+        "reference": "product_sale_badges",
+        "reference_default": "none"
       },
       {
         "type": "paragraph",
-        "content": "* the 'Product price label' is displayed when there is not a sale price."
+        "content": "* the 'Product price label' is displayed when there is not a sale price.",
```
Comment (junedkazi) [inline schema.json]: Won't we need force reload for type `paragraph` ?

Comment (kchu93) [reply schema.json]: I believe this is just a <p> element, and shouldn't need a force reload as we are not making any changes. @pedelman

File: schema.json (hunk 1 above)
Comment (kchu93) [reply schema.json]: 👍

Comment (junedkazi) [reply schema.json]: @karen-white @bookernath we will need to get all our theme developers to add `"force_reload": true` for all fields of `"type": "text"`

**Comment by Ubersmake** [comment]
💚 

I've tested one of the sections changed in `schema.json` and that's in order.

I also tried uploading a version without the changes in this PR, saving a configuration, and then _upgrading_ that theme to one with the changes in `schema.json`. The configuration changes that were saved were carried over into the upgraded version, even though the names and ids of two of the elements ("Global" and "Horizontal line") have changed.

**Pull request merged**